src/@types/File.ts

```diff
@@ -0,0 +1,27 @@
+export interface UrlFile {
+  type: 'url'
+
+  /**
+   * File index.
+   * @type {number}
+   */
+  index?: number
+
+  /**
+   * File URL.
+   * @type {string}
+   */
+  url?: string
```

Comment:
why is this nullable? can this exist without an url?

Reply:
You are not going to get the url when calling provider fileInfo :)

Reply:
@alexcos20 this is for encryption, `FileInfo` is what you get from provider.

Reply:
fixed, I was confused.  both url & method are required